src/scheduled-query/azext_scheduled_query/_help.py

```diff
@@ -30,14 +30,14 @@
                             [resource id RESOURCEID]
                             [where DIMENSION {includes,excludes} VALUE [or VALUE ...]
                             [and   DIMENSION {includes,excludes} VALUE [or VALUE ...] ...]]
-                            [at least MinTimeToFail out of EvaluationPeriod]'
+                            [at least MinTimeToFail violations out of EvaluationPeriod aggregated points]'
 
         Dimensions can be queried by adding the 'where' keyword and multiple dimensions can be queried by combining them with the 'and' keyword.
 examples:
   - name: Create a scheduled query for a VM.
-    text: az monitor scheduled-query create -g {rg} -n {name1} --scopes {vm_id} --condition "count \'union Event, Syslog | where TimeGenerated > ago(1h)\' > 360" --description "Test rule" --target-resource-type Microsoft.Compute/virtualMachines
+    text: az monitor scheduled-query create -g {rg} -n {name1} --scopes {vm_id} --condition "count 'union Event, Syslog | where TimeGenerated > ago(1h) | where EventLevelName==\"Error\" or SeverityLevel==\"err\"' > 360 at least 1 violations out of 5 aggregated points" --description "Test rule"
   - name: Create a scheduled query for VMs in a resource group.
-    text: az monitor scheduled-query create -g {rg} -n {name1} --scopes {rg_id} --condition "count \'union Event, Syslog | where TimeGenerated > ago(1h)\' < 260 at least 1 out of 5" --description "Test rule" --target-resource-type Microsoft.Compute/virtualMachines
+    text: az monitor scheduled-query create -g {rg} -n {name1} --scopes {rg_id} --condition "count 'union Event, Syslog | where TimeGenerated > ago(1h) | where EventLevelName==\"Error\" or SeverityLevel==\"err\"' > 360 at least 1 violations out of 5 aggregated points" --description "Test rule"
```

Comment:
This example should contain 'resource id _ResourceID' as it is set on a RG.

Reply:
what is `_ResourceID`? I tried to test it but failed. Could you give me an example?

Reply:
It is a column name that exists in most LA tables that contains the ID of the ARM resource that generated the log entry.

Reply:
It should work, what is the error you are getting?